keywording/text.xml

```diff
@@ -337,13 +337,20 @@ So packages are not to be marked stable for one of these architectures.
   <body>
     <p>
       If you maintain an architecture-independent package (data files,
-      icons, pure Perl,...) then you may request that your
+      icons, pure Python, ...) then you may request that your
       package be stabilized on all arches at once. To do this <d/> when
       you are filing the stabilization bug <d/> please add the keyword
       <c>ALLARCHES</c> in addition to <c>STABLEREQ</c> and CC the
       arches that you would like to stabilize.
     </p>
 
+    <p>
+      If your package is <c>ALLARCHES</c> material, you should
+      add the <c>&lt;stabilize-allarches&gt;</c> tag to metadata.xml.
+      This allows consistency in future stabilisations and saves
+      arch teams considerable work. Please be proactive in doing this.
```

Comment:
The last sentence isn't necessary. All the rationale has been given, so it is clear that the tag should be added to package metadata.

Reply:
Dropping.